subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -286,7 +286,7 @@ if BT_MESH_LIGHT_CTRL_REG
 
 config BT_MESH_LIGHT_CTRL_REG_SPEC
 	bool "Spec Lightness PI Regulator"
```

Comment:
We call it specification-defined illuminance regulator?

Reply:
Technically it is termed as "Light LC PI Feedback regulator". Documentation consistently uses Lightess PI Regulator I think.

Reply:
Sorry, I meant the spec. It think it is better to stay as is here because this is a spec term and this Kconfig option enables that. Changing is to illuminance regulator here might lead someone to think that this is a different thing than what is described in the spec. Probably rest of the documentation should be aligned to use proper terms for regulator. But that is a bigger task. If you agree, I will add it to backlog.

I suggest not making any changes here.